inference-engine/src/gna_plugin/frontend/weights_converter.hpp

```diff
@@ -50,15 +50,12 @@ inline bool convertWeights(InferenceEngine::CNNLayer* lp) {
     for (auto& dataItem : lp->outData) {
         dataItem->setPrecision(InferenceEngine::Precision::FP32);
     }
-    InferenceEngine::BlobMap newBlobs;
     for (auto& blob_pair : lp->blobs) {
         auto blob_name = blob_pair.first;
```

Comment:
Remove this unused variable before merging.

Reply:
done